02/app.js

```diff
@@ -20,4 +18,37 @@ function createRandomArray() {
 
 function getRandomInteger(min, max) {
     return Math.round(Math.random() * (max-min) + min);
-}
\ No newline at end of file
+}
+
+//task
+
+console.log('wyświetlenie w konsoli wszystkich elementów randomArray za pomocą pętli FOR')
+
+for (let i = 0; i < randomArray.length; i++) {
+    console.log(randomArray[i])
+}
+
+console.log('wyświetlenie w konsoli wszystkich elementów randomArray za pomocą metody tablicowej forEach')
+
+randomArray.forEach((el) => console.log(el));
+
+//https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
+console.log('Wyświetla elementy randomArray jako stringi')
+
+var writer = {
+    sb:       [],
+    write:    function (s) {
+        this.sb.push(s);
+    },
+    writeln:  function (s) {
+        this.write(s + "\n");
+    },
+    toString: function () {
+        return this.sb.join("");
+    }
+};
+
+randomArray.forEach(writer.writeln, writer);
+console.log(writer.toString());
```

Comment:
Przekombinowane, ale ok :P
Wiem, że to ćwiczenie i tutaj można sobie pozwolić jednak pamiętaj, aby w kodzie pisać jak najprostszy kod - taki potem jest najłatwiej utrzymać (rozwijać).

Reply:
to rozwiązanie pożyczyłem z //https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach, ale potem doszedłem do randomArray.forEach((el) => console.log(el)); i postanowiłem zostawić to pierwsze jako alternatywne (ale forEach jest dużo lepsze)